packages/muelu/test/scaling/MMKernelDriver.cpp

```diff
@@ -43,26 +43,159 @@
+#include "KokkosSparse_spgemm_handle.hpp"               // for StringToSPGEM...
+#include "KokkosSparse_spgemm_numeric.hpp"              // for spgemm_numeric
+#include "KokkosSparse_spgemm_symbolic.hpp"             // for spgemm_symbolic
+#include "Kokkos_ExecPolicy.hpp"                        // for RangePolicy
+#include "Kokkos_Macros.hpp"                            // for KOKKOS_LAMBDA
+#include "Kokkos_View.hpp"                              // for View
+#include "Teuchos_CommandLineProcessor.hpp"             // for CommandLinePr...
+#include "Teuchos_ENull.hpp"                            // for ENull::null
+#include "Teuchos_FancyOStream.hpp"                     // for FancyOStream
+#include "Teuchos_OrdinalTraits.hpp"                    // for OrdinalTraits
+#include "Teuchos_RCP.hpp"                              // for rcp_const_cast
+#include "Teuchos_RCPDecl.hpp"                          // for RCP, rcp
+#include "Teuchos_ScalarTraitsDecl.hpp"                 // for ScalarTraits
+#include "Tpetra_CrsMatrix_decl.hpp"                    // for CrsMatrix
+#include "Tpetra_Import_decl.hpp"                       // for Import
+#include <Tpetra_Import_Util2.hpp> // Import_Util::sortCrsEntries
+#include "Tpetra_Map_decl.hpp"                          // for Map
 #include <Xpetra_IO.hpp>
+#include "Xpetra_BlockedMap.hpp"                        // for LO, GO, NO
+#include "Xpetra_Map.hpp"                               // for UnderlyingLib
+#include "Xpetra_Parameters.hpp"                        // for Parameters
+namespace Teuchos { class ParameterList; }  // lines 92-92
+namespace Teuchos { class TimeMonitor; }  // lines 93-93
+namespace Teuchos { template <typename Ordinal> class Comm; }  // lines 94-94
+namespace Xpetra { template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node> class Matrix; }  // lines 95-95
+
+#ifdef HAVE_VTUNE
+#include "ittnotify.h"
+__itt_domain *        vtune_domain_modLTG;
+__itt_string_handle * vtune_task_modLTG_setup;
+__itt_string_handle * vtune_task_modLTG_mult;
+__itt_string_handle * vtune_task_modLTG_copy;
+#endif
 
-// Teuchos
-#include <Teuchos_StandardCatchMacros.hpp>
 
-// Galeri
-#include <Galeri_XpetraParameters.hpp>
-#include <Galeri_XpetraProblemFactory.hpp>
-#include <Galeri_XpetraUtils.hpp>
-#include <Galeri_XpetraMaps.hpp>
 // MueLu
 #include "MueLu.hpp"
 #include "MueLu_TestHelpers.hpp"
 
+#include <sys/types.h>
+#include <unistd.h>
+
+
+int getRssHWM ()
+{
+  static const std::string mem_file = "/proc/self/status";
+  std::ifstream infile(mem_file);
+
+  std::string line;
+  while (std::getline(infile, line))
+  {
+    if (line.rfind("VmHWM", 0) == 0) {
+      // VmHWM:      1764 kB
+      std::stringstream ss(line);
+      std::string key;
+      int value;
+      std::string unit;
+      ss >> key >> value >> unit;
+      infile.close();
+      return (value);
+    }
+  }
+  infile.close ();
+  return (-1);
+}
+
+#if USE_DESCRIPTIVE_STATS == 1
```

Comment:
Where does this macro get defined?  Macros need to be namespaced with their package name so that they don't collide with macros from other packages, TPLs, etc.